xbmc/music/dialogs/GUIDialogSongInfo.cpp

```diff
@@ -58,10 +58,13 @@ CGUIDialogSongInfo::CGUIDialogSongInfo(void)
   m_needsUpdate = false;
   m_startUserrating = -1;
   m_loadType = KEEP_IN_MEMORY;
+  m_songContributors = new CFileItemList;
```

Comment:
^that, and it's explicit who owns it. more reasoning here: https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rr-newdelete